ecl/hql/hqlusage.cpp

```diff
@@ -344,6 +344,7 @@ static IPropertyTree * addSelect(IPropertyTree * xml, IHqlExpression * expr, boo
     Owned<IPropertyTree> field = createPTree(isUsed ? "field" : "unused");
     field->setProp("@name", text.str());
     const char * tag = field->queryName();
+    xml = ensurePTree(xml, "fields");
```

Comment:
This is changed so that it can be easily mapped to cassandra tables using existing cassandra "mapping" code.